tests/GameTests.cpp

```diff
@@ -95,3 +95,97 @@ TEST_F(GameTests, score_should_be_three)
     ASSERT_EQ(3, checkValue);
 }
 
+TEST_F(GameTests, first_ball_should_be_3_and_second_4)
```

Comment:
test names are not proper. Please change them to something like this: GivenValue34ConvertValuesToPairOfIntsShouldReturn3And4